.github/workflows/realease-on-pr-merge.yaml

```
@@ -0,0 +1,47 @@
+        run: |
+          BUMP_TYPE="${{ steps.version_type.outputs.bump_type }}"
+          if [ "$BUMP_TYPE" = "major" [; then
+            npm version major
+          elif [ "$BUMP_TYPE" = "minor" ]; then
+            npm version minor
+          elif [ "$BUMP_TYPE" = "patch" ]; then
+            npm version patch
+          fi
+          git push origin HEAD
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v2
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)